docs/source/integrations/shaka.md

```diff
@@ -11,34 +11,35 @@ The [vuplay-shaka](https://github.com/Vualto/vuplay-shaka) repository demonstrat
 ```
 
 ```javascript
-var mpegdashStreamUrl = "<mpeg-dash-stream-url>";
-var vudrmToken = "<vudrm-token>";
+var mpegdashStreamUrl = "<your-stream-url>";
```

Comment:
Please change the name of the variable so that it matches the other integration docs.

`var streamUrl = "<your-stream-url>"`